app/views/layouts/application.html.erb

```diff
@@ -3,7 +3,7 @@
 <head>
   <title>GibctDataService</title>
   <%= stylesheet_link_tag    'application', media: 'all' %>
-  <%= javascript_include_tag 'application' %>
+  <%= javascript_include_tag 'application', nonce: '**CSP_NONCE**' %>
```

Comment:
[This](https://github.com/department-of-veterans-affairs/devops/blob/96861837aead268e79938695d5b839bd466f2514/ansible/deployment/config/revproxy-vagov/templates/nginx_revproxy.conf.j2#L328) is where we're doing the content swap in the reverse proxy.

Reply:
Can you point to the substitution logic?

Reply:
I would possibly coordinate with devops to see if there's a workaround for this. I'm seeing that the substitution takes place in the prod content security policy yml with some additional logic in the ngnix_revproxy config in the devops repo. An ops engineer may be more familiar

Reply:
I've talked with @zurbergram about this and once this is merged I will check staging to see if we are still seeing that CSP error in the browser console.

Reply:
Checked w/ @rianfowler and he's on board with that plan. This is a lot more front-end oriented than most of these PRs so a few more steps in the review chain